### PR TITLE
[RUMM-3306] Document react native SDK release 1.7.0

### PR DESCRIPTION
**Pull request by louiszawadzki**
### What does this PR do?
Adds documentation for the new features introduced in the React Native SDK version 1.7.0:

- Feature flag evaluation
- Upload frequency parameter
- Batch size parameter
- Background events tracking parameter

### Additional Notes

The release is not yet out, I will remove the "Do not merge" label once it is ready to be merged :) 

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/reactnative/advanced_configuration.md [hunk 1]
```diff
@@ -45,6 +45,165 @@ describe('App', () => {
...
+: Required<br/>
+**Type**: String<br/>
+The application's environment, for example: prod, pre-prod, and staging. Follows the [tag syntax requirements][15].
+
+`applicationId`
+: Required<br/>
+**Type**: String<br/>
+The RUM application ID.
+
+`trackInteractions`
```
[the remark this answers is not in the document]
Comment (louiszawadzki) [reply content/en/real_user_monitoring/reactnative/advanced_configuration.md:67]: copy/paste issue - it's deprecated for browser but not for React Native. I've removed the deprecation mention.

File: content/en/real_user_monitoring/reactnative/advanced_configuration.md [hunk 2]
```diff
@@ -45,6 +45,165 @@ describe('App', () => {
...
+: Optional<br/>
+**Type**: Boolean<br/>
+**Default**: `false` <br/>
+Enables automatic collection of users actions.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/reactnative/advanced_configuration.md:71]:
```suggestion
Enables automatic collection of user actions.
```

File: content/en/real_user_monitoring/reactnative/advanced_configuration.md [hunk 3]
```diff
@@ -45,6 +45,165 @@ describe('App', () => {
...
+
+`serviceName`
+: Optional<br/>
+**Type**: String<br/>
+The service name for your application. Follows the [tag syntax requirements][15].
+
+`version`
+: Optional<br/>
+**Type**: String<br/>
+The application's version, for example: 1.2.3, 6c44da20, and 2020.02.13. Follows the [tag syntax requirements][15].
```
Comment (rtrieu) [inline content/en/real_user_monitoring/reactnative/advanced_configuration.md:99]:
```suggestion
The application's version. For example: 1.2.3, 6c44da20, and 2020.02.13. Follows the [tag syntax requirements][15].
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some minor changes requested, but otherwise looks good!

File: content/en/real_user_monitoring/reactnative/_index.md [hunk 4]
```diff
@@ -328,6 +328,19 @@ DdSdkReactNative.setAttributes({
 });
 ```
 
+## Track background events
+
+<div class="alert alert-info"><p>Tracking background events may lead to additional sessions, which can impact billing. For questions, <a href="https://docs.datadoghq.com/help/">contact Datadog support.</a></p>
+</div>
+
+You can track events such as crashes and network requests when your application is in the background (for example, no active view is available).
```
[the remark this answers is not in the document]
Comment (louiszawadzki) [reply content/en/real_user_monitoring/reactnative/_index.md:336]: I'm not sure to understand your message. 
The way I understand it is that we should have "no active view is available" in the similar format it is displayed in RUM or in the app to the user. 

Actually we don't display any such information to users, when the a user switches app for instance, we stop RUM View tracking and therefore there is no active view for the SDK, but that information is not logged or displayed anywhere. 

I copied this part from the iOS / Android SDK docs: https://docs.datadoghq.com/real_user_monitoring/ios/?tab=swift#track-background-events

Comment (rtrieu) [inline content/en/real_user_monitoring/reactnative/_index.md:336]:
```suggestion
You can track events such as crashes and network requests when your application is in the background (for example, when no active view is available).
```

**Review by rtrieu — APPROVED** [review]
Looks good!

**Comment by louiszawadzki** [comment]
Thanks @rtrieu!
The PR is ready to be merged then :)

**Pull request merged**